src/sentry/auth/superuser.py

```diff
@@ -371,6 +384,10 @@ def enable_and_log_superuser_access():
             )
             enable_and_log_superuser_access()
         except AttributeError:
+            metrics.incr(
```

Comment:
Maybe rather than emitting all these different failures, we could just have a const `metric` at the top defaulted to success
Then each of these instances, we set `metric = 'superuser.failure'` and at the very end, incr the stat?

Reply:
yeah that works for me

Reply:
wait this might not work, on line `361` and `367` I raise an exception which means we can't incr the stat at the very end

Reply:
Another thought:

Maybe - we could differentiate between the different failure types with a reason tag?